services/director-v2/src/simcore_service_director_v2/models/schemas/image_meta.py

```diff
@@ -0,0 +1,116 @@
+"""
+    Implemented from jsonschema
+        api/specs/common/schemas/node-meta-v0.0.1.json
+        api/specs/common/schemas/node-meta-v0.0.1-converted.yaml
+
+    Schema of metadata injected in docker image labels of services
+"""
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, EmailStr, Field
+
+from ..constants import SERVICE_IMAGE_NAME_RE, VERSION_RE
+
+
+class Type(Enum):
+    computational = "computational"
+    dynamic = "dynamic"
```

Comment:
Should the left part not be in upper case? I forgot if there was a different reason to keep them lower.